### PR TITLE
Updated view.py, added utilities, exceptions and unit tests.

### PR DESCRIPTION
**Pull request by awielern**
Updated views.py, added utilities, exceptions and unit tests. Please review

### CODE REVIEW

File: CONVERT_SERVICE/convert_service/exception/convert_exception.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#
+# @views.py Copyright (c) 2021 Jalasoft.
```
Comment (MatiasBonifacio) [inline CONVERT_SERVICE/convert_service/exception/convert_exception.py]: Pls correct the copyright text

File: CONVERT_SERVICE/convert_service/exception/video_exception.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#
+# @views.py Copyright (c) 2021 Jalasoft.
```
Comment (MatiasBonifacio) [inline CONVERT_SERVICE/convert_service/exception/video_exception.py]: Pls correct the copyright text

File: CONVERT_SERVICE/convert_service/exception/image_exception.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#
+# @views.py Copyright (c) 2021 Jalasoft.
```
Comment (MatiasBonifacio) [inline CONVERT_SERVICE/convert_service/exception/image_exception.py]: Pls correct the copyright text

**Review by MatiasBonifacio — COMMENTED** [review]
Please correct the copyright text.

**Comment by awielern** [comment]
Copyright corrected

**Pull request closed without merging**